### ISSUE TITLE
Error installing on windows10

### ISSUE THREAD

**Issue by NJUShenbin** [docker/compose#4239]
I installed docker compose on windows by pip and it shows
```
Successfully installed docker-compose-1.9.0
```
But when I use docker compose command
```
PS F:\> docker-compose --version

//output
Microsoft.VC90.CRT.manifest could not be extracted!
```
I also download the docker-compose-Windows-x86_64.exe from https://github.com/docker/compose/releases.  Error is the same as above:
```
PS F:\> .\docker-compose-Windows-x86_64.exe

//output
Microsoft.VC90.CRT.manifest could not be extracted!
```
I can't find the solution on Google.It may be related to https://github.com/pyinstaller/pyinstaller/issues/1866.

**Comment by shin-** [comment]
Thanks for the report.

We're still waiting on a new release of PyInstaller, see here: https://github.com/docker/compose/pull/3439

**Comment by NJUShenbin** [comment]
Thank you very much!

**Comment by hellowangxuee** [comment]
Have you solve the problem? I got the same one and I can't find the solution.

**Comment by ijc** [comment]
I'm taking a punt that this old issue was solved via #3439 or some other change made since late 2016.
